internal/daemon/daemon.go

```diff
@@ -635,10 +635,6 @@ func (d *Daemon) Stop(sigCh chan<- os.Signal) error {
 	return nil
 }
 
-// This is a little more than servstate.killWait (but that isn't and shouldn't
-// be exported, so just duplicate it here).
-const stopRunningTimeout = 5*time.Second + 100*time.Millisecond
```

Comment:
That's where the 5 seconds came from, btw.  It's a reasonable time to be waiting for an action to get done before believing that it's actually dead.

Reply:
Thanks have adjusted below.